app/views/layouts/base.html.erb

```diff
@@ -67,6 +67,13 @@ See docs/COPYRIGHT.rdoc for more details.
         <% end %>
       <% end %>
     <% end %>
+    <% if ::I18n.locale == :lol #the in-context translation pseudo-language %>
+      <script type="text/javascript">
+        var _jipt = [];
+            _jipt.push(['project', 'openproject']);
```

Comment:
We might want to add the plugin projects here depending on their active state. Doesn't have to be made in this PR, however.

Reply:
Good catch. The original code did not provide this neither. However, I didn't find any documentation on how to add more crowdin projects. Maybe they do support it. Maybe not. Maybe it is as simple as adding another "push" as in line 73.